code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -1546,6 +1546,13 @@
 	M.update_transform()
 	..()
 
+/datum/reagent/plastic_polymers
+	name = "plastic polymers"
+	id = "plastic_polymers"
+	description = "the petroleum based components of plastic, contact a coder if you can actually see this" 
```

Comment:
I want to say you can see this in the chem master so probably not best to have this breaking the meta wall.